src/tests/unit/common/fail-test-on-error-logger.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { inspect } from 'util';
+import { Logger } from 'common/logging/logger';
+
+export const failTestOnErrorLogger: Logger = {
```

Comment:
You might have already handled it in part 2, I think https://github.com/microsoft/accessibility-insights-web/blob/ded30591606aeed2357977f33f38d28f7ce9f75a/src/tests/unit/tests/electron/platform/android/setup/state-machine/state-machine.test.ts#L210 will need to be updated to work with this

Reply:
Yes, I think you'd want to add a `noOpLogger` instead if you wanted something that didn't fail the test on errors for tests that intentionally introduce errors, or probably better, a mockLogger or a recordingLogger or something to validate what errors get logged